app/views/hyrax/homepage/_home_content.html.erb

```diff
@@ -1,75 +1,10 @@
-<h2><%= t('hyrax.homepage.admin_sets.title') %></h2>
-<div class="cards">
-  <% if @featured_work_list.empty? %>
-    <div class="card">
-      <h3><%= t('hyrax.homepage.featured_works.no_works') %></h3>
-    </div>
-  <% else %>
+<% if @featured_work_list.empty? %>
```

Comment:
Norm - this will read more naturally if you use

```
<% if @featured_work_list.featured_works.any? %>
    <% render ...partial %>
<% end %>
```

or 

```
<% unless featured_work_list.empty? %>
    <% render ...partial... %>
<% end %>
```

It would be clearest if `featured_work_list` implemented `.any?` itself, but it doesn't right now - so you have to do `featured_work_list.featured_works.any?`